### PR TITLE
CheckWitnessAnalyzer

### PR DESCRIPTION
**Pull request by Hecate2**

### CODE REVIEW

File: src/Neo.Compiler.CSharp/SecurityAnalyzer/CheckWitnessAnalyzer.cs [hunk 1]
```diff
@@ -0,0 +1,71 @@
...
+            {
+                this.droppedCheckWitnessResults = droppedCheckWitnessResults;
+                this.debugInfo = debugInfo;
+            }
+
+            public string GetWarningInfo(bool print = false)
+            {
+                if (droppedCheckWitnessResults.Count == 0)
+                    return "";
+                string result = $"[SEC] The returned values of CheckWitness at the following instruction addresses are DROPped:{Environment.NewLine}" +
```
Comment (Jim8y) [inline src/Neo.Compiler.CSharp/SecurityAnalyzer/CheckWitnessAnalyzer.cs:36]: DROPped?

**Review by Jim8y — APPROVED** [review]

File: src/Neo.Compiler.CSharp/SecurityAnalyzer/CheckWitnessAnalyzer.cs (hunk 1 above)
Comment (Hecate2) [reply src/Neo.Compiler.CSharp/SecurityAnalyzer/CheckWitnessAnalyzer.cs:36]: DROP after SYSCALL F8-27-EC-8C # System.Runtime.CheckWitness SysCall

Comment (Hecate2) [reply src/Neo.Compiler.CSharp/SecurityAnalyzer/CheckWitnessAnalyzer.cs:36]: I am emphasizing the instruction DROP. Can change if not lokking good.

File: src/Neo.Compiler.CSharp/SecurityAnalyzer/CheckWitnessAnalyzer.cs [hunk 2]
```diff
@@ -0,0 +1,71 @@
...
+                return result;
+            }
+        }
+
+        /// <summary>
+        /// 
+        /// </summary>
+        /// <param name="nef"></param>
+        /// <param name="manifest"></param>
+        /// <param name="debugInfo"></param>
```
Comment (shargon) [inline src/Neo.Compiler.CSharp/SecurityAnalyzer/CheckWitnessAnalyzer.cs:51]: This should be filled

Comment (Hecate2) [reply src/Neo.Compiler.CSharp/SecurityAnalyzer/CheckWitnessAnalyzer.cs:51]: > This should be filled

On it

Comment (Hecate2) [reply src/Neo.Compiler.CSharp/SecurityAnalyzer/CheckWitnessAnalyzer.cs:51]: https://github.com/neo-project/neo-devpack-dotnet/pull/1271

**Pull request merged**